docs/reference/installation.rst

```diff
@@ -237,18 +230,32 @@ Then add the following bundles in your `kernel::registerBundles()` method (after
                             SonataInvoiceBundle: ~
                             ApplicationSonataInvoiceBundle: ~
 
+Configure the bundles to use the newly generated classes
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+
+        sonata_customer:
+            class:
+                customer: App\Application\Sonata\CustomerBundle\Entity\Customer
+                address: App\Application\Sonata\CustomerBundle\Entity\Address
+                order: App\Application\Sonata\OrderBundle\Entity\Order
+                user: App\Application\Sonata\UserBundle\Entity\User
+
+
 Now, you can build up your database:
 
 .. code-block:: bash
 
-    $ app/console doctrine:schema:[create|update]
+    $ php bin/console doctrine:schema:update --force
 
 Create missing contexts:
 
 .. code-block:: bash
 
-    $ app/console sonata:classification:fix-context
-    $ app/console sonata:media:fix-media-context
+    $ php bin/console sonata:classification:fix-context
+    $ php app/console sonata:media:fix-media-context
```

Comment:
This needs to be the same as the line above

```suggestion
    $ php bin/console sonata:media:fix-media-context
```

Reply:
Must've overlooked that one. Thanks!